network-api/networkapi/wagtailpages/customblocks/tr_key_value_block.py

```diff
@@ -0,0 +1,22 @@
+from wagtail.core import blocks
+
+
+class TRKeyValueBlock(blocks.StructBlock):
+    """
+    This block is used for one-off page text blocks
+    in a way that allows for "easy" translation using
+    the wagtail-modeltranslation system of having the
+    translations be part of the CMS experience.
+    """
+
+    key = blocks.CharBlock(
+        required=True,
+    )
+
+    textblock = blocks.RichTextBlock(
+        required=True,
+        features=['bold', 'italic', 'h2', 'h3', 'h4', 'h5', 'h6', 'ol', 'ul', 'link'],
+    )
```

Comment:
I've made this text/richtext rather than text/text because I assumed our text would include things like links etc. and trying to do that in plain text is incredibly hard.

The downside is that, because of richtext, every text value is a `<p>...</p>` and I'm not sure how we feel about that.